corehq/apps/api/es.py

```diff
@@ -731,6 +733,6 @@ def es_search_by_params(search_params, domain, reserved_query_params=None):
 
     # add unconsumed filters
     for param, value in query_params.items():
-        payload["filter"]["and"].append(filters.term(param, value))
+        payload["filter"]["and"].append(filters.term(param, value.lower()))
```

Comment:
Can we add a note on why we do lower here?

Reply:
5e9000d9eea